float64 PosX
float64 PosY
float64 heading
